### ISSUE TITLE
About 99.2% accuracy

### ISSUE THREAD

**Issue by Em-Vi** [AnMol12499/CropcareAI#1]
Hello....It's Musthafa on this side

Hey...I became really interested in your code for plant diseases detection that was included in your CropCare ai project.

I wanted to integrate that AI into my website. I found out that the AI had some problems running in the older version of sklearn. So I decided to fine tune it using the newer version of sklearn. So as I was reading through your code in the notebook.I saw that you had achieved a 99.2% accuracy on the training samples. Isn't that overfitting?. I wanted to know how can I reduce the accuracy of the model to make it more adaptable AI. Do I wanna change the modelling or shouldni change the architecture.

I'll be looking forward for your reply and guidance on this issue
